xml/dhcp_vendor_class.xml

```diff
@@ -523,6 +523,14 @@
     <param pos="0" name="os.product" value="{hw.model} Firmware"/>
   </fingerprint>
 
+  <fingerprint pattern="^wyse-1000$">
+    <description>Dell Wyse 1000-series zero client</description>
+    <example>wyse-1000</example>
+    <param pos="0" name="hw.vendor" value="Dell"/>
+    <param pos="0" name="hw.device" value="Thin Client"/>
+    <param pos="0" name="hw.product" value="Wyse 1000 zero client"/>
```

Comment:
From a quick review of `h:dell:wyse_*` CPE strings I would suggest setting `hw.product` to `Wyse 1000` (9 contain simply the series/model number and 7 contain an additional string such as "thin_client"). There are also operating system CPE strings so I would add the following parameters:
```
<param pos="0" name="os.vendor" value="Dell"/>
<param pos="0" name="os.product" value="{hw.model} Firmware"/>
```